docs/features/analysis.md

```diff
@@ -920,3 +920,41 @@ data:
   api-key: <datadog-api-key>
   app-key: <datadog-app-key>
 ```
+
+## NewRelic Metrics
+
+A [New Relic](https://newrelic.com/) NRQL query can be used to obtain measurements for analysis.  
+
+```yaml
+apiVersion: argoproj.io/v1alpha1
+kind: AnalysisTemplate
+metadata:
+  name: success-rate
+spec:
+  args:
+  - name: application-name
+  metrics:
+  - name: success-rate
+    successCondition: result.successRate >= 0.95
+    provider:
+      newRelic:
+        profileSecretName: my-newrelic-secret  # optional, defaults to 'newrelic'
```

Comment:
I like the idea of profiles, which match the new relic CLI. However, I'd prefer to keep the fact that profiles are stored as secrets, an implementation detail from the spec. So can we rename this to just `profile` ?